pkg/runs/runsGet_test.go

```diff
@@ -141,18 +217,19 @@ func TestRunsGetOfRunNameWhichExistsProducesExpectedSummary(t *testing.T) {
 
 func TestRunsGetOfRunNameWhichDoesNotExistProducesError(t *testing.T) {
 	// Given ...
```

Comment:
Other tests: 
- Miss out the colon - should fail
- 2d:2w should fail, to time is bigger
- garbage
- :garbage
- :2w

Reply:
Also, you can test the detailed method, rather than with the whole request response stuff.

Not really testing much different each time otherwise.

Reply:
We coded these test up together.